### ISSUE TITLE
Potentially update clearfix util to be more modern

### ISSUE THREAD

**Issue by aarongeorge** [ismamz/postcss-utilities#27]
I've noticed that the clearfix util is using the `display: table/block` method, however there is a more modern approach which is to use `display: flow-root`.

A demo [is here](https://codepen.io/AaronGeorge/pen/YarKrm) showing it off.

**Comment by ismamz** [comment]
Hi @aarongeorge, I've think that in that case it's not absolutely necessary to use a mixin because you have a single property solution. A possible use case is when you need to refactor your codebase and migrate the `@util clearfix` to that new solution.  Also, [it's not fully cross-browser yet](https://caniuse.com/#feat=flow-root), but I'll keep this issue open to review it later.

**Comment by aarongeorge** [comment]
@ismamz fair enough mate, just thought I'd leave a comment on it anyway. You make a valid point around the individual prop though
